app/views/admin/items/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="new-form">
+  <%= form_for [:admin, @item] do |f| %>
+
+  <div class="form-group">
+    <%= f.label :title %>
+    <%= f.text_field :title %>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :price %>
+    <%= f.number_field :price %>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :description %>
+    <%= f.text_field :description %>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :image %>
+    <%= f.text_field :image %>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :status %>
+    <%= f.text_field :status %>
```

Comment:
@mmbensalah  In a refactor, we'd probably want to make this a drop down box that lists the two possible statuses for an item, that way we can ensure it gets recorded correctly as either 'active' or 'retired'.  I know Dylan used a dropdown like this in is trip view where you had to set the station.  This definitely works right now though!